CHANGELOG.md

```diff
@@ -111,39 +151,57 @@
 * Improved kart control at high speeds
 * Better placement of rescued karts
 * Transition track-making to blender 2.5/2.6
+### Tracks and modeling
+#### Karts
+* New Suzanne kart
+#### Tracks
+* Zen Garden by samuncle (replace Secret Garden)
+* New Subsea
+* New Island battle arena
+#### Miscellaneous
+* Added Thunderbird as race referee
 
 ## SuperTuxKart 0.7.2 (15. July 2011)
 * Added in-game addon manager
 * Fixed major memory leaks
-* New Snow Peak track by Samuncle
-* Improved star track UFO by Rudy
-* New Beastie kart.
 * Show when you get a highscore
 * Improve gamepad configuration under Windows (add ability to tell gamepads apart)
 * Various other tweaks done and glitches fixed
+### Tracks and modeling
+#### Karts
+* New Beastie kart.
+#### Tracks
+* Improved Snow Peak by samuncle
+* Improved Star Track UFO by Rudy
 
 ## SuperTuxKart 0.7.1b (21. April 2011)
 * Fix circular dependency in challenges
 * Updated translations
 
 ## SuperTuxKart 0.7.1 (15. April 2011)
 * Particle  (smoke, splash, fire) and weather effects
-* New Fort Magma by Samuncle, new Shiny Suburbs track by Horace
-* New Beagle kart by wolterh
 * Added internet news
 * Support for live language switch
 * Added optional minimal race UI
 * Temporary invincibility after being hit
 * Added support for full-screen anti-aliasing
 * Clearer multiplayer setup
 * Renamed many tracks to nicer names
+    * City become Shiny Suburb
+    * ?
```

Comment:
what is that?

Reply:
I wanted to document what were the name changes exactly. Seems I didn't finish it. It's a bit tiring to launch old versions to test those things.

I can remove those two lines for the time being and add it back when it's complete, if you wish.

Reply:
Yes please, I don't want to merge a question mark, that would be just weird. I;'d prefer it was not there, we can add it later if someone cares enough to dig through...

Reply:
Done